src/index.tsx

```diff
@@ -120,12 +129,16 @@ export class Pusher {
     pongTimeout?: Number;
     maxReconnectionAttempts?: Number;
     maxReconnectGapInSeconds?: Number;
+    authorizerTimeoutInSeconds?: Number;
     proxy?: string;
     onConnectionStateChange?: (
       currentState: string,
       previousState: string
     ) => void;
-    onAuthorizer?: (channelName: string, socketId: string) => any;
```

Comment:
`any` is very misleading here – if we e.g. `return undefined;`, the native callback won't be called through the bridge.

Better be explicit about what we require here – that's why I added the type

It's exported, so engineers using this SDK can import it and use in their app's code.